server/publications/channelAutocomplete.coffee

```diff
@@ -13,15 +13,12 @@ Meteor.publish 'channelAutocomplete', (name) ->
 		limit: 5
 
 	cursorHandle = RocketChat.models.Rooms.findByNameContainingAndTypes(name, ['c'], options).observeChanges
-		added: (_id, record) ->
-			pub.added('channel-autocomplete', _id, record)
-
+        added: (_id, record) ->
+			pub.added('channel-autocomplete', _id, record) if _id?
```

Comment:
Why would you check for _id? I wonder when does added, changed and removed not have the _id?